entries/p/payfit.com.json

```
@@ -0,0 +1,18 @@
+{
+  "Payfit": {
+    "domain": "payfit.com",
+    "url": "https://www.payfit.com",
+    "img": "payfit.com.png",
+    "documentation": "https://support.payfit.com/en/two-factor-authentication-for-administrators",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "keywords": [
+      "finance"
+    ],
+    "regions": [
+      "eu"
```

Looks like only the following countries are supported at the moment:
```suggestion
      "de",
      "es",
      "fr",
      "gb"
```